.circleci/README.md

```diff
@@ -104,7 +104,7 @@ All binaries are built in CircleCI workflows except Windows. There are checked-i
 
 Some quick vocab:
 
-* A \**workflow** is a CircleCI concept; it is a DAG of '**jobs**'. ctrl-f 'workflows' on https://github.com/pytorch/pytorch/blob/master/.circleci/config.yml to see the workflows.
```

Comment:
We actually might just want to delete this file. It's outdated and has not been updated in a while.

Reply:
Sounds good. I can do that.

Reply:
Deleted.